crypto/cmp/cmp_genm.c

```diff
@@ -344,3 +344,63 @@ int OSSL_CMP_get1_rootCaKeyUpdate(OSSL_CMP_CTX *ctx,
     X509_free(oldWithOld_copy);
     return res;
 }
+
+int OSSL_CMP_get1_crlUpdate(OSSL_CMP_CTX *ctx, const X509 *crlcert,
```

Comment:
where are unit or component tests for this?

Reply:
Tests are covered by `test/recipes/80-test_cmp_http_data/test_commands.csv`